### PR TITLE
zh: Further update TOC.md

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

Further update TOC.md

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):  https://github.com/pingcap/docs-dm/pull/75

### CODE REVIEW

File: zh/TOC.md [hunk 1]
```diff
@@ -9,20 +9,25 @@
...
-  - [主要特性](key-features.md)
-  - [应用场景](scenarios.md) 参考 [分表合并数据迁移最佳实践](shard-merge-best-practices.md)，[DM-worker 在上游 MySQL 主从间切换](usage-scenario-master-slave-switch.md)
+  + 主要特性
+    - [Table routing](key-features.md#table-routing)
+    - [Black & White Lists](key-features.md#black--white-table-lists)
+    - [Binlog Event Filter](key-features.md#binlog-event-filter)
+    - [同步延迟监控](key-features.md#同步延迟监控)
+    - [Online-ddl-scheme](feature-online-ddl-scheme.md)
+    - [分库分表合并同步](feature-shard-merge.md)
+    - [手动处理 Sharding DDL Lock](feature-manually-handling-sharding-ddl-locks.md)
```
Comment (csuzhangxc) [inline zh/TOC.md:19]: 这个作为特性好像不太恰当？

File: zh/TOC.md [hunk 2]
```diff
@@ -9,20 +9,25 @@
...
++ [应用场景](scenarios.md)
 + 快速上手
   - [部署集群](quick-start-with-dm.md)
-  - [同步任务](replicate-data-using-dm.md)
```
Comment (csuzhangxc) [inline zh/TOC.md:16]: 这个要不要还是放在这里，承接上面的『部署集群』？

Comment (yikeke) [reply zh/TOC.md:16]: 赞成

File: zh/TOC.md (hunk 1 above)
Comment (yikeke) [reply zh/TOC.md:19]: +1

**Comment by yikeke** [comment]
Please address comments. @toutdesuite 

PTAL @WangXiangUSTC @miaoqingli @lilin90

File: zh/TOC.md (hunk 1 above)
Comment (toutdesuite) [reply zh/TOC.md:19]: 放在『教程』部分是否适合呢？

File: zh/TOC.md (hunk 2 above)
Comment (toutdesuite) [reply zh/TOC.md:16]: updated

File: zh/TOC.md (hunk 1 above)
Comment (csuzhangxc) [reply zh/TOC.md]: 放『运维操作』里怎样呢？

Comment (toutdesuite) [reply zh/TOC.md]: updated in 3290dea4224138dcd900d6515d8b2c9a863b55b1~

File: zh/TOC.md [hunk 3]
```diff
@@ -36,6 +40,7 @@
     - 暂停 [参考](manage-replication-tasks.md#暂停数据同步任务)
     - 重启
     - 删除
+  - [手动处理 Sharding DDL Lock](feature-manually-handling-sharding-ddl-locks.md)
```
Comment (yikeke) [inline zh/TOC.md:43]:
```suggestion
  - [手动处理 Sharding DDL Lock](manually-handling-sharding-ddl-locks.md)
```
这个文件名更新下

**Pull request merged**